doc/sphinx/installation.rst

```diff
@@ -184,16 +184,26 @@ using the following commands:
     sudo rm -r ~/anaconda[23]
     sudo rm -r /Library/Python
 
+Install the command line tools (gcc, git, make, etc.) with either:
```

Comment:
I think we do not need to document this because it's already contained in the macports documentation https://www.macports.org/install.php

Reply:
Actually we do because we also need them for the Homebrew instructions. Also, we only tell people to download MacPorts, not to follow the install guide there.

Note that Xcode does not include gcc, so this line should mention either `clang` or the C++ compiler.